app/src/main/java/org/fossasia/openevent/app/utils/DateService.java

```diff
@@ -0,0 +1,53 @@
+package org.fossasia.openevent.app.utils;
+
+import org.fossasia.openevent.app.data.models.Event;
+
+import java.text.ParseException;
+import java.util.Date;
+
+public class DateService {
```

Comment:
>Distributing code among classes is opposite of making things messy

I was just suggesting to keep these two methods in event model as these are related to event model. No problem. :+1: